website/src/components/layout/sidebar.js

```diff
@@ -32,9 +32,9 @@ export const Sidebar = ({ items }) => {
 				gridColumnStart: 1,
 				gridColumnEnd: 2,
 				borderRight: `1px solid ${COLORS.border}`,
+				width: '300px',
```

Comment:
if we must have this, can we make it '100%' instead?

Reply:
This is to fix that overhang that I'm getting. I can't figure out how else to hide it. 100% makes it even wider. This is the only way I have found that fixes it.